.ci/pipeline/dr_weekly.yaml

```diff
@@ -0,0 +1,89 @@
+---
+job: LIBXLIO-dr-weekly-launcher
+failFast: false
+timeout_minutes: 120
+
+runs_on_agents:
+  - {nodeLabel: 'master', category: 'base'}
+
+env:
+  MAIL_FROM: jenkins@nvidia.com
+
+pipeline_start:
+  shell: action
+  module: groovy
+  run: |
+    if (!params.MAIL_TO.isEmpty()) {
+      currentBuild.displayName += "_weekly"
+      mail from: "${MAIL_FROM}",
+        mimeType: 'text/html',
+        to: "${MAIL_TO}",
+        subject: 'LibXLIO weekly DR build has been started',
+        body: """
+          <p><b>Build url:</b> <a href=${currentBuild.absoluteUrl}>link</a></p>"""
+    }
+
+steps:
+  - name: Determine release_tag
+    agentSelector:
+      - "{category: 'base'}"
+    shell: action
+    module: groovy
+    run: |
+      if (params.release_tag.isEmpty()) {
+        env.RELEASE_TAG = sh (returnStdout: true, script: "git describe --tags --abbrev=0 2>/dev/null").trim()
+      } else {
+        env.RELEASE_TAG = "${params.release_tag}"
+      }
+      echo "Release tag for build: ${env.RELEASE_TAG}"
+
+  - name: Check job state
+    shell: action
+    module: groovy
+    run: |
+      def job = Jenkins.instance.getItemByFullName('libxlio/LibXLIO-release-dr')
+      if (!job.isBuildable()) {
+          echo "The job '${env.JOB}' is disabled. Enabling..."
+          job.setDisabled(false)
+      } 
+      echo "The Job is enabled. Starting..."
+
+  - name: Run Job with parameters
+    shell: action
+    module: groovy
+    run: |
+      echo "Release tag we're building on: ${env.RELEASE_TAG}"
+      if (env.RELEASE_TAG) {
+        def build = build job: 'libxlio/LibXLIO-release-dr',
+          parameters: [
+            string(name: 'release_tag', value: RELEASE_TAG),
+            string(name: 'release_folder', value: "${release_folder}"),
+            string(name: 'notification_email', value: MAIL_TO),
+            string(name: 'sha1', value: sha1)
```

Comment:
Where do you set/define this parameter? this should point to vNext

Reply:
We do not define it, we determine release_tag and pass it to a child job. 
In a child job, the input parameter is:
- string:
            name: sha1
            default: 'tags/$release_tag'
In line 62, 'sha1' is set only for testing and is in the TEST_ONLY commit, which will be deleted before the merge. I need it for now because the changes have not been merged yet.